seaborn/distributions.py

```diff
@@ -129,6 +130,27 @@ def distplot(a, bins=None, hist=True, kde=True, rug=False, fit=None,
         >>> from scipy.stats import norm
         >>> ax = sns.distplot(x, fit=norm, kde=False)
 
+    Plot the distribution with a histogram, a KDE and fitted sum of 2
+    gaussians. This is a numerical algorithm (scipy.optimize.curve_fit), so a
+    reasonable initial guess of the parameters (p0 passed through fit_kws) is
+    important.
+
+    .. plot::
+        :context: close-figs
+
+        >>> def gauss(x, *p0):
+        ...     a, p0 = p0[0], p0[1:]
+        ...     return a*norm.pdf(x, *p0)
+        >>> def gauss2(x, *p0):
+        ...     p1 = p0[:2]
+        ...     p2 = p0[2:]
+        ...     return gauss(x, *p1)+gauss(x, *p2)
+        >>> a = np.append(np.random.normal(0.0, 0.5, size=500),
```

Comment:
Alternatively - you can add command continuation characters (`...`):
https://wiki.python.org/moin/MultiLineStringsInDocTest